jwst/extract_2d/extract_2d.py

```diff
@@ -11,18 +11,20 @@
 log.setLevel(logging.DEBUG)
 
 
-def extract2d(input_model,
-              slit_names=None,
-              source_ids=None,
-              reference_files={},
-              grism_objects=None,
-              tsgrism_extract_height=None,
-              wfss_extract_half_height=None,
-              extract_orders=None,
-              mmag_extract=None,
-              nbright=None):
+def extract2d(
+    input_model,
+    slit_names=None,
+    source_ids=None,
+    reference_files=None,
+    grism_objects=None,
+    tsgrism_extract_height=None,
+    wfss_extract_half_height=None,
+    extract_orders=None,
+    mmag_extract=None,
+    nbright=None,
+):
     """
-    The main extract_2d function
+    Perform the main extract_2d function.
```

Comment:
Can this be changed to say what this is actually doing, e.g., "extract 2-d cutouts from ..."